presto-main/src/test/java/com/facebook/presto/operator/scalar/FunctionAssertions.java

```diff
@@ -576,7 +576,6 @@ else if (type == Block[].class) {
 
         List<Object> results = new ArrayList<>();
 
-        //
```

Comment:
For the future reference - try to minimize the number of unrelated changes you introduce.